hack/verify-dependencies.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Waiting for somebody picking this up as a side- or hackaton project: go-check-imports, with dot files specifiying the allowed and forbidden imports for the subtree.

Until that happens... bash.